RELEASE.md

```diff
@@ -108,21 +113,15 @@ Signing a tag with a GPG key is appreciated, but in case you can't add a GPG key
 
 Once a tag is created, the release process through CircleCI will be triggered for this tag and Circle CI will draft the GitHub release using the `prombot` account.
 
-Now all you can do is to wait for tarballs to be uploaded to the Github release and the container images to be pushed to the Docker Hub and Quay.io. Once that has happened, click _Publish release_, which will make the release publicly visible and create a GitHub notification.
+Finally, wait for the build step for the tag to finish. The point here is to wait for tarballs to be uploaded to the Github release and the container images to be pushed to the Docker Hub and Quay.io. Once that has happened, click _Publish release_, which will make the release publicly visible and create a GitHub notification.
+** Note: ** for a release candidate version the _This is a pre-release_ box when drafting the release in the Github UI.
```

Comment:
** Note: ** If the release is for a release candidate, check the _This is a pre-release_ box in the GitHub UI before clicking _Publish release_.

Reply:
The CI job takes care of creating the release for you and check the pre-release flag if necessary.

Reply:
Even better.

Reply:
So this should probably just read `Ensure the _pre-release_ box is checked before clicking _Publish_ on the release page.`?